azimuth/types/model_performance.py

```diff
@@ -30,18 +34,46 @@ class UtteranceCountPerFilterValue(AliasModel):
     filter_value: str = Field(..., title="Filter value")
 
 
-class UtteranceCountPerFilter(AliasModel):
-    prediction: Optional[List[UtteranceCountPerFilterValue]] = Field(
-        ..., title="Prediction", nullable=True
+if typing.TYPE_CHECKING:
+
+    class GenericAliasModel(AliasModel, Generic[T]):
+        pass
+
+    class GenericAliasModelPipeline(AliasModel, Generic[T]):
+        pass
+
+    ValuePerDatasetSmartTag = GenericAliasModel
+    ValuePerPipelineSmartTag = GenericAliasModelPipeline
```

Comment:
Was there a problem when using the same class twice?
```suggestion
    ValuePerPipelineSmartTag = GenericAliasModel
```

Reply:
Yes, Utterance inherits from both so they have to be different types.